class-wxr-importer.php

```diff
@@ -1568,7 +1574,22 @@ protected function process_author( $data, $meta ) {
 			$user_id
 		) );
 
-		// TODO: Implement meta handling once WXR includes it
+		// Update user meta.
+		foreach ( $meta as $meta_item ) {
+			$key = $meta_item['key'] ?? false;
+			$value = $meta_item['value'] ?? false;
+
+			// export gets meta straight from the DB so could have a serialized string
+			if ( ! $value ) {
+				$value = maybe_unserialize( $value );
+			}
+
+			if ( ! empty( $key ) ) {
```

Comment:
**question:** Should we add `all` user meta fields? Some fields might have been used as `flags` (no value using if-exists checks). Secondly, the current `empty()` check will not add values that are `0`.